Tasks/SonarQubePostTest/PRCA/Orchestrator.ps1

```diff
@@ -41,15 +41,19 @@ function GetCommentsFromIssues
     Write-Verbose "Transforming SonarQube analysis issues to PR comments"
     
     $comments = New-Object System.Collections.ArrayList
-
     foreach ($issue in $issues)
     {
         $priority = GetCommentPriority $issue
 
-        Assert ( ![String]::IsNullOrWhiteSpace($issue.message) ) "Internal error: the SonarQube reported issues do not have a property named 'message'"
-        Assert ( ![String]::IsNullOrWhiteSpace($issue.line) ) "Internal error: the SonarQube reported issues do not have a property named 'line'"        
-        Assert ( ![String]::IsNullOrWhiteSpace($issue.relativePath) ) "Internal error: the SonarQube reported issues do not have a property named 'relativePath'"
-        Assert ( ![String]::IsNullOrWhiteSpace($issue.rule) ) "Internal error: the SonarQube reported issues do not have a property named 'rule'"
+        if ([String]::IsNullOrWhiteSpace($issue.line) -or $issue.line -lt 0)
```

Comment:
$issue.line -lt 0 means "if the line is smaller than 0", i.e. negative line numbers. A pretty unusual scenario, it could be treated as an exception instead...